opencti-platform/opencti-front/src/private/components/data/taxii/TaxiiCollectionCreation.tsx

```diff
@@ -88,6 +89,12 @@ const TaxiiCollectionCreation: FunctionComponent<TaxiiCollectionCreationProps> =
   const { t_i18n } = useFormatter();
   const classes = useStyles();
   const [filters, helpers] = useFiltersState(emptyFilterGroup);
+  const [areInferenceRulesResultAdded, setAreInferenceRulesResultAdded] = useState(true);
```

Comment:
why is `include_inferrences` in a separate state and not directly part of the form ?

Reply:
updated ! :) 
thanks !